draft-ietf-jsonpath-base.md

```diff
@@ -183,12 +183,14 @@ Value:
   composed of components such as structured values, namely JSON objects and arrays, and
   primitive data, namely numbers and text strings as well as the special
   values null, true, and false.
+  {{-json}} focuses on the textual representation of JSON values and
+  doesn't define the value abstraction assumed here.
 
 Member:
 : A name/value pair in an object.  (A member is not itself a value.)
 
 Name:
-: The name in a name/value pair constituting a member.   (A name is not itself a value.) <!-- should we make it clear that names are (Unicode) strings? -->
+: The name (a string) in a name/value pair constituting a member.
   This is also used in {{-json}}, but that specification does not
```

Comment:
Worse, Section 3 defines "false", "true" and "null" as "literal names".
These cannot be used in a member name position!

Reply:
That's a shame. As you wrote earlier, let's not try and fix JSON here. But if you were suggesting we say something about this, please say so.